lib/couchbase/bucket.rb

```diff
@@ -134,13 +134,5 @@ def ping(options = Options::Ping::DEFAULT)
         end
       end
     end
-
-    # @api private
-    # TODO: deprecate in 3.1
-    PingOptions = ::Couchbase::Options::Ping
```

Comment:
Can we delete these, or should we just mark them as deprecated for now?

Reply:
@DemetrisChr I have fixed it. Now `lib/couchbase/deprecations.rb` intercepts undefined constants and can substitute correct one while emitting warning.